### PR TITLE
[Serializer] Deprecate annotations

### LINKED ISSUE #19036: [FrameworkBundle][Serializer] Deprecate annotations
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#51470
| PR author(s) | @alexandre-daubois
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/19036

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -279,6 +279,13 @@ for each format:
 
     $classMetadataFactory = new ClassMetadataFactory(new AnnotationLoader(new AnnotationReader()));
```
Comment (OskarStark) [inline components/serializer.rst:280]: should be removed?

Comment (OskarStark) [reply components/serializer.rst:280]: or remove it in 7.0?

Comment (alexandre-daubois) [reply components/serializer.rst:280]: It is still valid in 6.4 so I think it _should_ stay with the deprecation message below. If someone wants to migrate from one to another, this person may search for `AnnotationLoader` in the doc to understand how to migrate

I would remove it in 7.0 :slightly_smiling_face:

Comment (OskarStark) [reply components/serializer.rst:280]: Can you open a PR?

**Comment by OskarStark** [comment]
Thank you Alexandre.

File: components/serializer.rst (hunk 1 above)
Comment (alexandre-daubois) [reply components/serializer.rst:280]: Yep 🙂

**Pull request merged**